content/ansible-tower-getting-started/2-cred/_index.md

```diff
@@ -43,21 +43,22 @@ node3 ansible_host=<IP>
 ansible ansible_host=<IP>
 ```
 
-Note the names for the nodes and the IP addresses, we will use them to fill the inventory in Tower now:
+But to make your life easier there is a `/etc/hosts` file that will resolve the nodes `node1`, `node2` and `node3` to their IP addresses. So you can just use the short hostnames.
 
-  - In the inventory view of Tower click on your **Workshop Inventory**
+Now create the inventory in Tower:
+
+  - In the inventory view of the Tower web UI click on your **Workshop Inventory**
 
   - Click on  the **HOSTS** button
 
   - To the right click the ![plus](../../images/green_plus.png?classes=inline) button.
 
   - **HOST NAME:** node1
-
-  - **Variables:** Under the three dashes `---`, enter **ansible_host: 22.33.44.55** in a new line. Make sure to enter your specific IP address for your **node1** from the inventory looked up above, and note that the variable definition has a colon **:** and a space between the values, not an equal sign **=** like in the inventory file.
+  <!-- - **Variables:** Under the three dashes `---`, enter **ansible_host: 22.33.44.55** in a new line. Make sure to enter your specific IP address for your **node1** from the inventory looked up above, and note that the variable definition has a colon **:** and a space between the values, not an equal sign **=** like in the inventory file. -->
```

Comment:
is this supposed to be a multi line comment? I guess the <!-- was added by mistake?

Reply:
Deleted